integration-tests/validation/src/testDebug/kotlin/com/android/designcompose/testapp/validation/RenderAllExamples.kt

```diff
@@ -44,7 +45,7 @@ import org.robolectric.annotation.GraphicsMode
 // Enable Robolectric Native Graphics (RNG)
 @GraphicsMode(GraphicsMode.Mode.NATIVE)
 @RunWith(ParameterizedRobolectricTestRunner::class)
-@Config(qualifiers = "w1920dp-h1500dp-xlarge-long-notround-any-xhdpi-keyshidden-nonav")
+@Config(qualifiers = "w1920dp-h1500dp-xlarge-long-notround-any-xhdpi-keyshidden-nonav", sdk = [33])
```

Comment:
We want this for sdk 33 or higher, right? So you should uses [minsdk](https://robolectric.org/configuring/#configure-sdk-level)

Reply:
And also, why do we need to set the minSDK if the test is disabled? I realized this is RenderAllExamples, we want this to cover all of our supported SDK versions, so we shouldn't be setting sdk restrictions on it.